### ISSUE TITLE
Uninitialized TermDisplay error

### ISSUE THREAD

**Issue by mdedetrich** [olafurpg/scalafmt#691]
- **Version**: 0.5.2
- **Integration**: sbt
- **System**: MacOsx 10.12.2

When running scalafmt 0.5.2 on any of my projects, get the following error. Have tried it in various shells(bash/sh/fish) and Terminal applications (including the basic MacOsx Terminal)

```
[scully] $ scalafmt
Downloading scalafmt artifacts...
org.scalafmt.bootstrap.FetchError: ArrayBuffer((Dependency(com.geirsson:scalafmt-cli_2.11,0.5.2,compile,Set(),Attributes(jar,),false,true),List(not found: /Users/mdedetrich/.ivy2/local/com.geirsson/scalafmt-cli_2.11/0.5.2/ivys/ivy.xml, download error: Caught java.lang.Exception: Uninitialized TermDisplay (Uninitialized TermDisplay))))
	at org.scalafmt.bootstrap.ScalafmtBootstrap$.fromVersionUncached(Bootstrap.scala:69)
	at org.scalafmt.bootstrap.ScalafmtBootstrap$$anonfun$fromVersion$1.apply(Bootstrap.scala:45)
	at org.scalafmt.bootstrap.ScalafmtBootstrap$$anonfun$fromVersion$1.apply(Bootstrap.scala:45)
	at scala.collection.mutable.MapLike$class.getOrElseUpdate(MapLike.scala:189)
	at scala.collection.mutable.AbstractMap.getOrElseUpdate(Map.scala:91)
	at org.scalafmt.bootstrap.ScalafmtBootstrap$.fromVersion(Bootstrap.scala:45)
	at org.scalafmt.bootstrap.ScalafmtBootstrap$.main(Bootstrap.scala:93)
	at org.scalafmt.sbt.ScalafmtPlugin$autoImport$$anonfun$scalafmt$1.apply(ScalafmtPlugin.scala:15)
	at org.scalafmt.sbt.ScalafmtPlugin$autoImport$$anonfun$scalafmt$1.apply(ScalafmtPlugin.scala:12)
	at sbt.Command$$anonfun$sbt$Command$$apply1$1$$anonfun$apply$6.apply(Command.scala:71)
	at sbt.Command$.process(Command.scala:93)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:96)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:96)
	at sbt.State$$anon$1.process(State.scala:184)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:96)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:96)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.MainLoop$.next(MainLoop.scala:96)
	at sbt.MainLoop$.run(MainLoop.scala:89)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:68)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:63)
	at sbt.Using.apply(Using.scala:24)
	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:63)
	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:46)
	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:30)
	at sbt.MainLoop$.runLogged(MainLoop.scala:22)
	at sbt.StandardMain$.runManaged(Main.scala:57)
	at sbt.xMain.run(Main.scala:29)
	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
	at xsbt.boot.Launch$.run(Launch.scala:109)
	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
	at xsbt.boot.Launch$.launch(Launch.scala:117)
	at xsbt.boot.Launch$.apply(Launch.scala:18)
	at xsbt.boot.Boot$.runImpl(Boot.scala:41)
	at xsbt.boot.Boot$.main(Boot.scala:17)
	at xsbt.boot.Boot.main(Boot.scala)
[error] org.scalafmt.bootstrap.FetchError: ArrayBuffer((Dependency(com.geirsson:scalafmt-cli_2.11,0.5.2,compile,Set(),Attributes(jar,),false,true),List(not found: /Users/mdedetrich/.ivy2/local/com.geirsson/scalafmt-cli_2.11/0.5.2/ivys/ivy.xml, download error: Caught java.lang.Exception: Uninitialized TermDisplay (Uninitialized TermDisplay))))
```

**Comment by olafurpg** [comment]
I'm unable to reproduce, but I've merged a potential fix and published it as 0.5.3-RC1 (still syncing to maven). Please confirm if it works.

**Comment by mdedetrich** [comment]
@olafurpg Yup, 0.5.3-RC1 works

**Comment by olafurpg** [comment]
Great, I published the fix as 0.5.3, thanks for reporting!
